test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -276,12 +276,8 @@ def author(value):
             raise Invalid("Invalid author")
 
 
-def doc_schema(module_name, version_added=True):
-    deprecated_module = False
+def doc_schema(module_name, version_added=True, deprecated_module = False):
 
-    if module_name.startswith('_'):
-        module_name = module_name[1:]
```

Comment:
Isn't this still needed (for core)?

Reply:
I pass in`deprecated_module` to this function now.

Reply:
Yes, but the modified `module_name` is needed in `doc_schema_dict`.

Reply:
ah, yes, reverted delete

Reply:
The `deprecated_module = True` can still go I think.